curl/Dockerfile

```diff
@@ -1,3 +1,3 @@
 FROM alpine:3.10
 
-RUN apk --no-cache --update add curl=7.66.0-r0
+RUN apk --no-cache --update add curl=~7
```

Comment:
why no use [official curl images](https://hub.docker.com/r/curlimages/curl) ?

Reply:
Legacy security reason...
Why not using the official one :)
Nobody is stopping you to do so right ;)